client/src/App.js

```diff
@@ -19,10 +19,12 @@ export default class App extends Component {
   };
 
   render() {
+      console.log("this.state.savedlist", this.state.savedList);    
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList}></Route>
```

Comment:
Well composed, is there anything with this that you want to do but aren't sure yet regarding routes?